src/components/tabs/sell.tab.tsx

```diff
@@ -52,7 +52,11 @@ function SellTabContent({ needsUserDataForm }: { needsUserDataForm: boolean }):
 
   useEffect(() => {
     if (!sellableAssets) return;
-    Promise.all(sellableAssets.map((asset) => readBalance(asset, address))).then(setAssetBalances);
+    Promise.all(sellableAssets.map((asset) => readBalance(asset, address)))
+      .then((balances) => balances.sort((a, b) => b.balance.minus(a.balance).toNumber()))
+      .then((balances) => balances.sort((a, b) => (a.asset.sortOrder ?? Infinity) - (b.asset.sortOrder ?? Infinity)))
```

Comment:
Are you sure this is working? Isn't the second sorting "overwriting” the first one?

Reply:
hm... might just be that test data and the wanted result matched, but yes thinking about it, it will just overwrite it. will take a look again.